hs/src/Reach/Eval/Core.hs

```diff
@@ -2984,14 +2984,9 @@ evalPrim p sargs =
       ensure_after_first
       zero_args
       evalPrim (SLPrim_fluid_read FV_didSend) []
-    SLPrim_Unstrict -> do
+    SLPrim_unstrict -> do
       one_arg >>= \case
-        SLV_Clo clo_at Nothing clo@(SLClo _ clo_args _ _) -> do
-          at <- withAt id
-          unless (null clo_args) $
-            expect_thrown clo_at $
-              Err_Apply_ArgCount at 0 (length clo_args)
-          locUseUnstrict True $ evalApply (SLV_Clo clo_at Nothing clo) []
+        clo@SLV_Clo {} -> locUseUnstrict True $ evalApply clo []
```

Comment:
I don't even think you should do that... just

```
x <- one_arg
locUseUnstrict .... x ...
```

`evalApply` will error if it needs to